packages/harpguru-core/package.json

```diff
@@ -23,25 +23,6 @@
     "react-native-redash": "^14.1.1",
```

Comment:
This does not appear to be the case following a fresh install. Perhaps it *was* because of the version number being incorrect, but even during that time it didn't seem to be having an impact. See the README.md for instructions on how we will aim to keep this version number up to date.